Upgrade Manubot Rootstock on 2018-12-18

dhimmel opened the pull request:
Includes enhanced citation support.

dhimmel committed:
Enable ISBN citation support

Merges https://github.com/greenelab/manubot-rootstock/pull/149

* Enable ISBN citaiton support

* Update URL citation
Previous URL citation contained "?", therefore requiring a citation tag.

dhimmel committed:
webpage utility: upgrade preexisting timestamps

Merges https://github.com/greenelab/manubot-rootstock/pull/150

Attempt to upgrade preexisting timestamps during continuous
deployment, such that manually upgrading incomplete timestamps
should no longer be necessary.

Add two mutually exclusive arguments to webpage.py:
--ots-cache and --no-ots-cache for configuring the OTS client
caching.

Disable timestamp caching for deploy.sh (i.e. on Travis CI)

dhimmel committed:
Upgrades on 2018-12-18 including wikidata citation support

Merges https://github.com/greenelab/manubot-rootstock/pull/151

* Upgrade manubot python package with Wikidata citations
* Remove DataCite DOI from 02.delete-me.md
* Add additional manuscripts to USAGE examples

dhimmel committed:
Update Manubot Rootstock version on 2018-12-18

agitter reviewed (approved):
I'm approving the changes, but do we need to wait for datacite to come back online before merging?  Or we could merge this and immediately switch to a manual or URL-based citation for that reference in a new pull request.

dhimmel commented:
`@url:https://zietzm.github.io/Vagelos2017/` will not work because there is both a translation-server issue at https://github.com/zotero/translation-server/issues/65 and Greycite is down.

I'll just set the doi metadata in manual references in this PR, using one of the past output references.json as a template.

dhimmel committed:
Use manual reference for DataCite DOI

DataCite's DOI Content Negotiation backend is down. See
https://github.com/greenelab/manubot/issues/89

agitter commented:
> I'll just set the doi metadata in manual references in this PR, using one of the past output references.json as a template.

Looks good to me.
